drawBot/drawBotDrawingTools.py

```diff
@@ -1933,6 +1934,8 @@ def imageSize(self, path, pageNumber=None):
             # its an NSImage
             rep = path
         else:
+            if isinstance(path, pathlib.Path):
+                path = f'{path}'
```

Comment:
1. To convert a single object to a string one uses `str(x)`, not an f-string that only contains a single placeholder
2. However, the correct way to convert a Path object to a string path is `os.fspath(p)`
3. You need to do `isinstance(path, os.PathLike)` instead of checking agains `pathlib.Path`

Reply:
Also, see comment below: change the `if isinstance(path, str)` line below to `if isinstance(path, (str, os.PathLike))` and remove the elif.